app/Models/Product.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Product extends Model
+{
+    use HasFactory;
+    use SoftDeletes;
+
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
+        'title',
+        'description',
+        'price',
+        'discount',
+        'photo',
+        'category_id',
+    ];
+
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array<string, string>
+     */
+
+     public function category()
+    {
+        return $this->belongsTo(Category::class);
+    }
+
+    public function orders()
+    {
+        return $this->belongsToMany(Order::class, 'order_products')
+                    ->withPivot('quantity', 'price')
```

Comment:
ეს რას შვება აღმიწერე აქვე რეფლაიდ, და შენით დაწერე თუ არა ბარემ ესეც ჰაჰა

Reply:
ამას public function orderProducts(): HasMany
    {
        return $this->hasMany(OrderProduct::class);
    }
    :'''')